pixelpact/src/compare.js

```diff
@@ -5,19 +5,21 @@ export async function compare(expected, actual, options = { threshold: 0.01 }) {
   const expectedPng = PNG.sync.read(expected);
```

Comment:
As we destruct `expectedPng` few lines below, we could already use object destructuring here: `const {width, height, data} = expectedPng;`